fairlearn/metrics/__init__.py

```diff
@@ -6,12 +6,17 @@
 from .group_metric_result import GroupMetricResult  # noqa: F401
 from .metrics_engine import metric_by_group, make_group_metric  # noqa: F401
 
+from .extra_metrics import specificity_score, miss_score, fallout_score
+
 # Classification metrics
 group_accuracy_score = make_group_metric(skm.accuracy_score)
 group_confusion_matrix = make_group_metric(skm.confusion_matrix)
 group_precision_score = make_group_metric(skm.precision_score)
```

Comment:
Now I look at these again, I have a nasty feeling that wrapping the `scikit-learn` metrics like this is implicitly restricting their use to binary classification on {0, 1}